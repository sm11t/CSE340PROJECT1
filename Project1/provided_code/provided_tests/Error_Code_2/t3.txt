TASKS
1 2
POLY
    F1 = x - 2;
    F2(Y,Z) = x (x + 2)^2 + 
    (5 + (Y + V)^2 ) +
    K H I B^2 Z;
EXECUTE
    a = F1(2);
    OUTPUT a;
INPUTS
    3 5 6