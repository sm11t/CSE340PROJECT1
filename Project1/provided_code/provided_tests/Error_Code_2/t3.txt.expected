Semantic Error Code 2: 5 5 6 7 7 7 7 
